### ISSUE TITLE
[Broken] TF1.fr multi-language videos: no detection of other languages than French

### ISSUE THREAD

**Issue by jzoghaib** [yt-dlp/yt-dlp#982]
## Checklist

- [x] I'm reporting a broken site support
- [x] I've verified that I'm running yt-dlp version **2021.09.02**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
yt-dlp -v -F https://www.tf1.fr/tf1/get-the-gringo/videos/kill-the-gringo-86959372.html
[debug] Command-line config: ['-v', '-F', 'https://www.tf1.fr/tf1/get-the-gringo/videos/kill-the-gringo-86959372.html']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] yt-dlp version 2021.09.02 (zip)
[debug] Python version 3.7.3 (CPython 64bit) - Linux-5.10.0-0.bpo.5-amd64-x86_64-with-debian-10.10
[debug] exe versions: ffmpeg 4.1.6-1, ffprobe 4.1.6-1, phantomjs 2.1.1, rtmpdump 2.4
[debug] Optional libraries: mutagen, sqlite
[debug] Proxy map: {}
[debug] [TF1] Extracting URL: https://www.tf1.fr/tf1/get-the-gringo/videos/kill-the-gringo-86959372.html
[TF1] kill-the-gringo-86959372: Downloading JSON metadata
[debug] [wat.tv] Extracting URL: wat:13802773
[wat.tv] 13802773: Downloading JSON metadata
[wat.tv] 13802773: Downloading MPD manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13802773:
ID                    EXT RESOLUTION FPS |   TBR PROTO | VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
--------------------- --- ---------- --- - ----- ----- - ----------- ----- --------- ---- ------- --------------------------
dash-audio_fra=64000  m4a audio only     |   64k dash  |                   mp4a.40.2  64k 48000Hz [fr], DASH audio, m4a_dash
dash-audio_fra=128000 m4a audio only     |  128k dash  |                   mp4a.40.2 128k 48000Hz [fr], DASH audio, m4a_dash
dash-video=200033     mp4 416x234    25  |  200k dash  | avc1.42C01E  200k                        DASH video, mp4_dash
dash-video=400072     mp4 480x270    25  |  400k dash  | avc1.42C01E  400k                        DASH video, mp4_dash
dash-video=600100     mp4 640x360    25  |  600k dash  | avc1.42C01E  600k                        DASH video, mp4_dash
dash-video=1200222    mp4 1024x576   25  | 1200k dash  | avc1.4D401F 1200k                        DASH video, mp4_dash
dash-video=1700265    mp4 1024x576   25  | 1700k dash  | avc1.4D401F 1700k                        DASH video, mp4_dash
dash-video=2500406    mp4 1280x720   25  | 2500k dash  | avc1.4D401F 2500k                        DASH video, mp4_dash

yt-dlp -v -F "https://das-q1-ssl.tf1.fr/2/USP-0x0/27/73/13802773/ssm/6a2603d6515db50912cfb89b775467977b553a21ca27ae131180d424ecab3a73.ism/13802773.mpd?e=1631733432&max_bitrate=2700000&st=uIejdE925vcQ9GqvWS9w_Q"
[debug] Command-line config: ['-v', '-F', 'https://das-q1-ssl.tf1.fr/2/USP-0x0/27/73/13802773/ssm/6a2603d6515db50912cfb89b775467977b553a21ca27ae131180d424ecab3a73.ism/13802773.mpd?e=1631733432&max_bitrate=2700000&st=uIejdE925vcQ9GqvWS9w_Q']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] yt-dlp version 2021.09.02 (zip)
[debug] Python version 3.7.3 (CPython 64bit) - Linux-5.10.0-0.bpo.5-amd64-x86_64-with-debian-10.10
[debug] exe versions: ffmpeg 4.1.6-1, ffprobe 4.1.6-1, phantomjs 2.1.1, rtmpdump 2.4
[debug] Optional libraries: mutagen, sqlite
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://das-q1-ssl.tf1.fr/2/USP-0x0/27/73/13802773/ssm/6a2603d6515db50912cfb89b775467977b553a21ca27ae131180d424ecab3a73.ism/13802773.mpd?e=1631733432&max_bitrate=2700000&st=uIejdE925vcQ9GqvWS9w_Q
[generic] 13802773: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] 13802773: Downloading webpage
[generic] 13802773: Extracting information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13802773:
ID               EXT RESOLUTION FPS |   TBR PROTO | VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
---------------- --- ---------- --- - ----- ----- - ----------- ----- --------- ---- ------- --------------------------
audio_eng=64000  m4a audio only     |   64k dash  |                   mp4a.40.2  64k 48000Hz [en], DASH audio, m4a_dash
audio_fra=64000  m4a audio only     |   64k dash  |                   mp4a.40.2  64k 48000Hz [fr], DASH audio, m4a_dash
audio_eng=128000 m4a audio only     |  128k dash  |                   mp4a.40.2 128k 48000Hz [en], DASH audio, m4a_dash
audio_fra=128000 m4a audio only     |  128k dash  |                   mp4a.40.2 128k 48000Hz [fr], DASH audio, m4a_dash
video=200033     mp4 416x234    25  |  200k dash  | avc1.42C01E  200k                        DASH video, mp4_dash
video=400072     mp4 480x270    25  |  400k dash  | avc1.42C01E  400k                        DASH video, mp4_dash
video=600100     mp4 640x360    25  |  600k dash  | avc1.42C01E  600k                        DASH video, mp4_dash
video=1200222    mp4 1024x576   25  | 1200k dash  | avc1.4D401F 1200k                        DASH video, mp4_dash
video=1700265    mp4 1024x576   25  | 1700k dash  | avc1.4D401F 1700k                        DASH video, mp4_dash
video=2500406    mp4 1280x720   25  | 2500k dash  | avc1.4D401F 2500k                        DASH video, mp4_dash

```

## Description

On TF1.fr some videos are available with original audio in addition to the French dub, the latter being the default audio when playing the video in a browser.
However, yt-dlp does not currently detect any other audio streams than the default French, as shown in the first part of the log for the example video located at "https://www.tf1.fr/tf1/get-the-gringo/videos/kill-the-gringo-86959372.html" since all the currently detected audio streams are identified with "audio_fra=XXX" IDs whereas this video also has English audio.

I suspect that the problem is related to the detection or analysis of the mpd manifest of the video since English streams can be detected (and downloaded) when using yt-dlp with the url of the mpd manifest. Indeed, using the browser inspector and especially the network analysis, I have identified that the mpd manifest of the example video is located at "https://das-q1-ssl.tf1.fr/2/USP-0x0/27/73/13802773/ssm/6a2603d6515db50912cfb89b775467977b553a21ca27ae131180d424ecab3a73.ism/13802773.mpd?e=1631733432&max_bitrate=2700000&st=uIejdE925vcQ9GqvWS9w_Q". As shown in the second part of the log, when using yt-dlp with the mpd manifest url, original audio streams (in English) are detected and identified with "audio_eng=XXX" IDs. There is then no problem to download them when using the mpd url.

Additional information on TF1.fr:
- an account may be needed in order to watch the videos with a browser but creating this account only requires providing an email address and password. The account is not needed when using yt-dlp.
- videos are usually geo-restricted to France, as a consequence the use of a proxy may be needed to work on the issue outside of France
- many videos are time-limited (usually 7 days) when they are provided as part of the TV-catchup service but the example video above (and others I could provide if needed) should not have a time limit (or a very long one).

**Comment by mavosaure** [comment]
I confirm the @jzoghaib (very detailed) report, but also #1683  reported by @JustALooker666:
yt-dlp currently forget foreign languages and subtitles on tf1.fr

If I seek the dash/mpd file using Firefox inspector and I use it directly with yt-dlp, this one, using generic extractor (instead of [TF1] or [wat.tv] with the main URL), I can have both languages audio files and also subtitles files.

```
 yt-dlp -F -v  https://www.tf1.fr/tf1/les-tudors/videos/les-tudors-s01-e01-la-paix-universelle-96887813.html
[debug] Command-line config: ['-F', '-v', 'https://www.tf1.fr/tf1/les-tudors/videos/les-tudors-s01-e01-la-paix-universelle-96887813.html']
[debug] User config file: /home/user/.config/yt-dlp/config
[debug] User config: ['-o', '~/Vidéos/TV/%(title)s.%(ext)s', '--all-subs', '--add-metadata', '--xattrs', '--embed-subs', '--convert-subs', 'srt', '--embed-thumbnail']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1 (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite
[debug] Proxy map: {}
[debug] [TF1] Extracting URL: https://www.tf1.fr/tf1/les-tudors/videos/les-tudors-s01-e01-la-paix-universelle-96887813.html
[TF1] les-tudors-s01-e01-la-paix-universelle-96887813: Downloading JSON metadata
[debug] [wat.tv] Extracting URL: wat:13825481
[wat.tv] 13825481: Downloading JSON metadata
[wat.tv] 13825481: Downloading MPD manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13825481:
ID                    EXT RESOLUTION FPS │  FILESIZE    TBR PROTO │ VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
dash-audio_fra=64000  m4a audio only     │ ~25.05MiB    64k dash  │                   mp4a.40.2  64k 48000Hz [fr], DASH audio, m4a_dash
dash-audio_fra=128000 m4a audio only     │ ~50.09MiB   128k dash  │                   mp4a.40.2 128k 48000Hz [fr], DASH audio, m4a_dash
dash-video=200057     mp4 416x234     25 │ ~78.29MiB   200k dash  │ avc1.42C01E  200k                        DASH video, mp4_dash
dash-video=400091     mp4 480x270     25 │ ~156.58MiB  400k dash  │ avc1.42C01E  400k                        DASH video, mp4_dash
dash-video=600132     mp4 640x360     25 │ ~234.87MiB  600k dash  │ avc1.42C01E  600k                        DASH video, mp4_dash
dash-video=1200597    mp4 1024x576    25 │ ~469.86MiB 1200k dash  │ avc1.4D401F 1200k                        DASH video, mp4_dash
dash-video=1700799    mp4 1024x576    25 │ ~665.62MiB 1700k dash  │ avc1.4D401F 1700k                        DASH video, mp4_dash
dash-video=2501141    mp4 1280x720    25 │ ~978.84MiB 2501k dash  │ avc1.4D401F 2501k                        DASH video, mp4_dash

 yt-dlp -v -F "https://das-q1-ssl.tf1.fr/2/USP-0x0/54/81/13825481/ssm/9614545665a0b261362ee8fbb2de363704b7adb7f09bcefab97911c1fb7cf0c0.ism/13825481.mpd?e=1637554124&max_bitrate=2700000&st=oaeoI-KVU3hUydPONwyzEA"
[debug] Command-line config: ['-v', '-F', 'https://das-q1-ssl.tf1.fr/2/USP-0x0/54/81/13825481/ssm/9614545665a0b261362ee8fbb2de363704b7adb7f09bcefab97911c1fb7cf0c0.ism/13825481.mpd?e=1637554124&max_bitrate=2700000&st=oaeoI-KVU3hUydPONwyzEA']
[debug] User config file: /home/user/.config/yt-dlp/config
[debug] User config: ['-o', '~/Vidéos/TV/%(title)s.%(ext)s', '--all-subs', '--add-metadata', '--xattrs', '--embed-subs', '--convert-subs', 'srt', '--embed-thumbnail']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1 (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://das-q1-ssl.tf1.fr/2/USP-0x0/54/81/13825481/ssm/9614545665a0b261362ee8fbb2de363704b7adb7f09bcefab97911c1fb7cf0c0.ism/13825481.mpd?e=1637554124&max_bitrate=2700000&st=oaeoI-KVU3hUydPONwyzEA
[generic] 13825481: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] 13825481: Downloading webpage
[generic] 13825481: Extracting information
[debug] Identified a DASH manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Downloading subtitles: fr
[info] Available formats for 13825481:
ID               EXT RESOLUTION FPS │   TBR PROTO │ VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
audio_eng=64000  m4a audio only     │   64k dash  │                   mp4a.40.2  64k 48000Hz [en], DASH audio, m4a_dash
audio_fra=64000  m4a audio only     │   64k dash  │                   mp4a.40.2  64k 48000Hz [fr], DASH audio, m4a_dash
audio_eng=128000 m4a audio only     │  128k dash  │                   mp4a.40.2 128k 48000Hz [en], DASH audio, m4a_dash
audio_fra=128000 m4a audio only     │  128k dash  │                   mp4a.40.2 128k 48000Hz [fr], DASH audio, m4a_dash
video=200057     mp4 416x234     25 │  200k dash  │ avc1.42C01E  200k                        DASH video, mp4_dash
video=400091     mp4 480x270     25 │  400k dash  │ avc1.42C01E  400k                        DASH video, mp4_dash
video=600132     mp4 640x360     25 │  600k dash  │ avc1.42C01E  600k                        DASH video, mp4_dash
video=1200597    mp4 1024x576    25 │ 1200k dash  │ avc1.4D401F 1200k                        DASH video, mp4_dash
video=1700799    mp4 1024x576    25 │ 1700k dash  │ avc1.4D401F 1700k                        DASH video, mp4_dash
video=2501141    mp4 1280x720    25 │ 2501k dash  │ avc1.4D401F 2501k                        DASH video, mp4_dash
```
Could somebody help us to fix the extractor?

**Comment by mavosaure** [comment]
Maybe there's some refactoring to do as:
- **tf1** extractor refers to wat.tv extractor
- **wat** extractor refers to [the late platform wate.tv operated by the French TV channel TF1](http://www.wat.tv/) until its closure in [2016](https://fr.wikipedia.org/wiki/Wat.tv)
- **lci** extractor refers to wat.tv, as LCI French channel is also owned by TF1
- LCI uses the same kind of MPD/DASH file as TF1


```
yt-dlp -v -F https://www.lci.fr/replay-lci/video-anti-complot-l-emission-du-05-11-2021-2201090.html -F
[debug] Command-line config: ['-v', '-F', 'https://www.lci.fr/replay-lci/video-anti-complot-l-emission-du-05-11-2021-2201090.html', '-F']
[debug] User config file: /home/user/.config/yt-dlp/config
[debug] User config: ['-o', '~/Vidéos/TV/%(title)s.%(ext)s', '--all-subs', '--add-metadata', '--xattrs', '--embed-subs', '--convert-subs', 'srt', '--embed-thumbnail']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1 (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite
[debug] Proxy map: {}
[debug] [LCI] Extracting URL: https://www.lci.fr/replay-lci/video-anti-complot-l-emission-du-05-11-2021-2201090.html
[LCI] 2201090: Downloading webpage
[debug] [wat.tv] Extracting URL: wat:13825415
[wat.tv] 13825415: Downloading JSON metadata
[wat.tv] 13825415: Downloading MPD manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13825415:
ID                 EXT RESOLUTION FPS │  FILESIZE    TBR PROTO │ VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
dash-audio=64000   m4a audio only     │ ~24.47MiB    64k dash  │                   mp4a.40.2  64k 48000Hz DASH audio, m4a_dash
dash-audio=128000  m4a audio only     │ ~48.94MiB   128k dash  │                   mp4a.40.2 128k 48000Hz DASH audio, m4a_dash
dash-video=200082  mp4 416x234     25 │ ~76.50MiB   200k dash  │ avc1.42C01E  200k                        DASH video, mp4_dash
dash-video=400168  mp4 480x270     25 │ ~152.99MiB  400k dash  │ avc1.42C01E  400k                        DASH video, mp4_dash
dash-video=600230  mp4 640x360     25 │ ~229.48MiB  600k dash  │ avc1.42C01E  600k                        DASH video, mp4_dash
dash-video=1200371 mp4 1024x576    25 │ ~458.93MiB 1200k dash  │ avc1.4D401F 1200k                        DASH video, mp4_dash
dash-video=1700485 mp4 1024x576    25 │ ~650.14MiB 1700k dash  │ avc1.4D401F 1700k                        DASH video, mp4_dash
dash-video=2500681 mp4 1280x720    25 │ ~956.07MiB 2500k dash  │ avc1.4D401F 2500k                        DASH video, mp4_dash

yt-dlp -F -v "https://das-q1-ssl.tf1.fr/2/USP-0x0/54/15/13825415/ssm/13825415.ism/13825415.mpd?e=1637561089&max_bitrate=2000000&st=Twvs-ERCqUVPhygE90blGQ"
[debug] Command-line config: ['-F', '-v', 'https://das-q1-ssl.tf1.fr/2/USP-0x0/54/15/13825415/ssm/13825415.ism/13825415.mpd?e=1637561089&max_bitrate=2000000&st=Twvs-ERCqUVPhygE90blGQ']
[debug] User config file: /home/user/.config/yt-dlp/config
[debug] User config: ['-o', '~/Vidéos/TV/%(title)s.%(ext)s', '--all-subs', '--add-metadata', '--xattrs', '--embed-subs', '--convert-subs', 'srt', '--embed-thumbnail']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1 (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://das-q1-ssl.tf1.fr/2/USP-0x0/54/15/13825415/ssm/13825415.ism/13825415.mpd?e=1637561089&max_bitrate=2000000&st=Twvs-ERCqUVPhygE90blGQ
[generic] 13825415: Requesting header
WARNING: [generic] Falling back on generic information extractor.
[generic] 13825415: Downloading webpage
[generic] 13825415: Extracting information
[debug] Identified a DASH manifest
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13825415:
ID            EXT RESOLUTION FPS │   TBR PROTO │ VCODEC        VBR ACODEC     ABR  ASR    MORE INFO
───────────────────────────────────────────────────────────────────────────────────────────────────────────────
audio=64000   m4a audio only     │   64k dash  │                   mp4a.40.2  64k 48000Hz DASH audio, m4a_dash
audio=128000  m4a audio only     │  128k dash  │                   mp4a.40.2 128k 48000Hz DASH audio, m4a_dash
video=200082  mp4 416x234     25 │  200k dash  │ avc1.42C01E  200k                        DASH video, mp4_dash
video=400168  mp4 480x270     25 │  400k dash  │ avc1.42C01E  400k                        DASH video, mp4_dash
video=600230  mp4 640x360     25 │  600k dash  │ avc1.42C01E  600k                        DASH video, mp4_dash
video=1200371 mp4 1024x576    25 │ 1200k dash  │ avc1.4D401F 1200k                        DASH video, mp4_dash
video=1700485 mp4 1024x576    25 │ 1700k dash  │ avc1.4D401F 1700k                        DASH video, mp4_dash

```
But maybe it could be also a better idea to open a new issue...

**Comment by JustALooker666** [comment]
I do hope someone will fix it, especially ripping subtitles. 
I for one use an online video grabber and I can get vids with a higher bitrate from the master m3u8 including 1080p.
yt-dlp always fixes "the malformed AAC stream" but had no problems playing any video I've download this way.

```
yt-dlp -v -F https://das-q1-ssl.tf1.fr/2/USP-0x0/77/12/13827712/ssm/13827712.ism/13827712.m3u8
[debug] Command-line config: ['-v', '-F', 'https://das-q1-ssl.tf1.fr/2/USP-0x0/77/12/13827712/ssm/13827712.ism/13827712.m3u8']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2021.11.10.1
[debug] Python version 3.8.9 (CPython 64bit) - Linux-4.15.0-161-generic-x86_64-with-glibc2.17
[debug] exe versions: ffmpeg 4.4-static (setts), ffprobe 4.4-static, phantomjs ., rtmpdump 2.4
[debug] Optional libraries: Cryptodome, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [generic] Extracting URL: https://das-q1-ssl.tf1.fr/2/USP-0x0/77/12/13827712/ssm/13827712.ism/13827712.m3u8
[generic] 13827712: Requesting header
[debug] Identified a direct video link
[generic] 13827712: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[info] Available formats for 13827712:
ID   EXT RESOLUTION FPS │   TBR PROTO  │ VCODEC        VBR ACODEC     ABR
──────────────────────────────────────────────────────────────────────────
280  mp4 416x234     25 │  280k m3u8_n │ avc1.42C01E  200k mp4a.40.2  64k
560  mp4 480x270     25 │  560k m3u8_n │ avc1.42C01E  400k mp4a.40.2 128k
772  mp4 640x360     25 │  772k m3u8_n │ avc1.42C01E  600k mp4a.40.2 128k
1409 mp4 1024x576    25 │ 1409k m3u8_n │ avc1.4D401F 1200k mp4a.40.2 128k
1939 mp4 1024x576    25 │ 1939k m3u8_n │ avc1.4D401F 1700k mp4a.40.2 128k
2787 mp4 1280x720    25 │ 2787k m3u8_n │ avc1.4D401F 2501k mp4a.40.2 128k
3848 mp4 1280x720    25 │ 3848k m3u8_n │ avc1.64001F 3501k mp4a.40.2 128k
5438 mp4 1920x1080   25 │ 5438k m3u8_n │ avc1.640028 5001k mp4a.40.2 128k
```
